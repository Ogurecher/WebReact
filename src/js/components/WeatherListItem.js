import React, { Component } from "react";

export default class WeatherListItem extends Component {
    constructor(props) {
        super(props);

        this.state = {value: 'WeatherListItem'};
    }
    
    render() {
        return (
            <p>
                {this.props.items}
            </p>
        );
    }
}