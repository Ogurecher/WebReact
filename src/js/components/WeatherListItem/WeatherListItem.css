.city-info-weather-list-item {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    background: grey;
    color: white;
    margin-top: 1px;
    padding: 1%;
}