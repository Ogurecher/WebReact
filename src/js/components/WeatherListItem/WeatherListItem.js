import React, { Component } from 'react';
import './WeatherListItem.css'

export default class WeatherListItem extends Component {

    render() {
        return (
            <div className='city-info-weather-list-item'>
                <p> {this.props.label} </p>
                <p> {this.props.content} </p>
            </div>
            
        );
    }
}