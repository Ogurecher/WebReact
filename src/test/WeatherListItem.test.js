import React from 'react';
import renderer from 'react-test-renderer';

import WeatherListItem from '../js/components/WeatherListItem/WeatherListItem';

describe('WeatherListItem', function() {
    test('snapshot renders', function() {
        const labelStub = 'Wind';
        const contentStub = [`10deg`, `; 10 m/s`];

        const component = renderer.create(<WeatherListItem label={labelStub} content={contentStub} />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});