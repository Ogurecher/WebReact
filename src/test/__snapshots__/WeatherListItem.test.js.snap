// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherListItem snapshot renders 1`] = `
<div
  className="city-info-weather-list-item"
>
  <p>
     
    Wind
     
  </p>
  <p>
     
    10deg
    ; 10 m/s
     
  </p>
</div>
`;
